[DOC][HOLD] describe redaction modes and allowed attributes for traces LBAC (#122841)
grafana/grafana: docs/sources/administration/data-source-management/teamlbac/configure-teamlbac-for-tempo/index.md

## How LBAC affects returned data

> Grafana Cloud Traces supports three redaction modes that control how unauthorized spans are handled in trace by ID search responses. The active mode is configured per tenant. To change the mode for your organization, contact Grafana Support.

Reviewer: We usually use the full product name for the first occurance, so here we can use Cloud Traces instead.

Reviewer: ```suggestion
Cloud Traces supports three redaction modes that control how unauthorized spans are handled in trace by ID search responses. The active mode is configured per tenant. To change the mode for your organization, contact Grafana Support.
```

### Attributes mode (default)

Non-matching spans are included in the response but have their attributes and intrinsics redacted. Only the following minimal fields remain visible on redacted spans:

- `traceId`, `spanId`, `parentId`
- `name`, `kind`
- `timestamps`
- `status`

> You can extend the set of always-visible fields by configuring `allowed_attributes` — a list of scoped attributes and intrinsics such as `span:name`, `resource.service.name`, `event:name`, or `scope.version` that are never redacted, even from non-matching spans. Contact Grafana Support to configure `allowed_attributes` for your organization.

Reviewer: ```suggestion
You can extend the set of always-visible fields by configuring `allowed_attributes`, which sets a list of scoped attributes and intrinsics that are never redacted, For example, you can set `span:name`, `resource.service.name`, `event:name`, or `scope.version` so they are never redacted, even from non-matching spans. Contact Grafana Support to configure `allowed_attributes` for your organization.
```
